config/web/settings.ini

```diff
@@ -10,16 +10,20 @@ web.port=8080
 web.max_threads=100
 
 # HTTPS/SSL configuration
+
 web.https=false
+
+# WARNING: the following HTTPS default values are for testing purposes only!
+# do not use them in production but create your own keystore, etc.
+
 # path to keystore, can be absolute or relative to SCHEDULER_HOME
-# WARNING: default value is for testing purposes only!
-# do not use it in production but create your own
-web.https.keystore=config/web/keystore
-web.https.keystore.password=activeeon
+#web.https.keystore=config/web/keystore
+#web.https.keystore.password=activeeon
+#web.https.allow_any_hostname=false
```

Comment:
@fviale Thank you for the comment but please wait, I haven't asked for reviews yet ;)

Usually, I do a new pass on the changes before and after Jenkins the Jenkins build has passed. Consequently, it is possible that lot of changes still occur before I ask for review.